src/load.py

```diff
@@ -35,16 +41,13 @@ def lambda_handler(event, context):
     :param context:
     :return: success or fail status
     """
+    response = None
     try:
         logger.debug(event)
-        etl(event)
-        return {
-            'statusCode': 200,
-            'body': 'Successfully processed S3 Event'
-        }
+        record_ids = etl(event)
     except Exception as e:
         logger.info(f'About to exit with this exception: {repr(e)}')
-        return {
-            'statusCode': 500,
-            'body': 'Error processing S3 Event %s' % repr(e)
-        }
+        raise e
+    else:
+        response = {'ids': record_ids}
+    return response
```

Comment:
Returning of list of record IDs because an AWS S3 Event can in principle contain more than one object.